test/acceptance/schema/update_class_test.go

```diff
@@ -68,3 +68,114 @@ func TestUpdateClassDescription(t *testing.T) {
 		assert.Equal(t, res.Payload.Description, newDescription)
 	})
 }
+
+func TestUpdatePropertyDescription(t *testing.T) {
+	className := "C2"
+	propName := "p1"
+	nestedPropName := "np1"
+
+	delete := func() {
+		params := clschema.NewSchemaObjectsDeleteParams().WithClassName(className)
+		_, err := helper.Client(t).Schema.SchemaObjectsDelete(params, nil)
+		assert.Nil(t, err)
+	}
+	defer delete()
+
+	func() {
```

Comment:
Why are we using func(){--code--}() instead of just --code-- directly?

Reply:
I like closures for marking clearly separated units of work/logic but am happy to get rid